app/src/androidTest/java/ch/epfl/sdp/UserInfectionTest.java

```diff
@@ -79,9 +80,12 @@ public boolean updateStatus(Carrier.InfectionStatus stat) {
         };
         fragment.getLocationService().setAnalyst(analyst);
         receiver = fragment.getLocationService().getReceiver();
+        fragment.getActivity().getSharedPreferences("UserInfectionPrefFile", Context.MODE_PRIVATE)
```

Comment:
Could you please me explain what is "Context.MODE_PRIVATE"?

Reply:
It just means that the preference object is not shared with other applications. So it is stored inside the app private folder